src/components/DiffView/index.spec.tsx

```diff
@@ -741,4 +854,45 @@ describe(__filename, () => {
       ).toEqual(false);
     });
   });
+
+  describe('trimHunkChanges', () => {
+    it('does not trim when not necessary', () => {
+      const hunks = [
+        createInternalHunkWithChanges([{ content: '// example content 1' }]),
+        createInternalHunkWithChanges([{ content: '// example content 2' }]),
+      ];
+
+      expect(trimHunkChanges(hunks, { maxLength: 2 })).toEqual(hunks);
+    });
+
+    it('trims the hunk changes when too long', () => {
+      const hunks = [
+        createInternalHunkWithChanges([{ content: '// example content 1' }]),
+        createInternalHunkWithChanges([{ content: '// example content 2' }]),
+        createInternalHunkWithChanges([{ content: '// example content 3' }]),
+      ];
+
+      expect(
+        getAllHunkChanges(trimHunkChanges(hunks, { maxLength: 2 })),
+      ).toHaveLength(2);
```

Comment:
Would it be worth asserting that the two hunks included are the ones we expect to be there?